packages/slate-editor/src/components/OptionsGroup/Option.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import * as React from 'react';
+
+import { uniqueId } from '#lodash';
+
+import styles from './OptionsGroup.module.scss';
+
+export interface OptionsGroupOption<T extends string> {
+    label: string;
+    value: T;
+    Icon?: React.ComponentType<React.SVGProps<SVGSVGElement> & { isActive: boolean }>;
+    disabled?: boolean;
+}
+
+export interface ChooseOptionProps<T extends string> {
+    name: string;
+    option: OptionsGroupOption<T>;
+    selected?: T;
+    onChange: (value: T | undefined) => void;
```

Comment:
Why do we have `undefined` in `onChange` here? Under which conditions it can produce `onChange(undefined)`?